bug_monitor_add.php

```diff
@@ -85,13 +91,15 @@
 	$g_project_override = $t_bug->project_id;
 }
 
-if( $t_logged_in_user_id == $t_user_id ) {
-	access_ensure_bug_level( config_get( 'monitor_bug_threshold' ), $f_bug_id );
-} else {
-	access_ensure_bug_level( config_get( 'monitor_add_others_bug_threshold' ), $f_bug_id );
-}
+foreach( $t_user_ids as $t_user_id ) {
+	if( $t_logged_in_user_id == $t_user_id ) {
+		access_ensure_bug_level( config_get( 'monitor_bug_threshold' ), $f_bug_id );
+	} else {
+		access_ensure_bug_level( config_get( 'monitor_add_others_bug_threshold' ), $f_bug_id );
+	}
 
-bug_monitor( $f_bug_id, $t_user_id );
+	bug_monitor( $f_bug_id, $t_user_id );
+}
 
 form_security_purge( 'bug_monitor_add' );
```

Comment:
Give the amount of possible errors above, I wonder if this call should be earlier on this page.  Not part of this change, but should be easy to change.